### PR TITLE
[record-logs-on-model] Tee logs from flow onto model

### PR DESCRIPTION
**Pull request by wlonk**
cat

### CODE REVIEW

**Comment by wlonk** [comment]
@cdcarter I'm not getting the values I expect from teeing the logs here, and I'd love to talk it over with you, at the least as a rubber duck, possibly to sort out any surprises I'm missing. Thanks!

**Comment by wlonk** [comment]
@cdcarter If you'd take another look at this, I'd appreciate it. What else do you think this needs?

**Comment by wlonk** [comment]
@cdcarter and now I've added log scrubbing per https://trello.com/c/yXylAywZ/88-anonymize-cci-log

File: metadeploy/api/belvedere_utils.py [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+    text = obscure_salesforce_org_name(text)
+    return text
+
+
+def obscure_mpinstaller_deployment_test_failure(text):
+    """
+    Returns 'Apex Test Failure' as the error text if the text contains a test failure
+    message.
+    """
+    if text.find("Apex Test Failure: ") != -1:
```
Comment (davisagli) [inline metadeploy/api/belvedere_utils.py]:
```suggestion
    if "Apex Test Failure: " in text:
```

**Pull request merged**